var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -101,7 +101,7 @@ def patch(self):
 
     @property
     def common_config_args(self):
-        return [
+        config_args = [
             '-prefix', self.prefix,
```

Comment:
If you can figure out which options aren't supported by qt3, we can move these out of the common_config_args and into qt4/5 specific ones.